src/Storages/Statistics/ConditionSelectivityEstimator.cpp

```diff
@@ -9,43 +11,67 @@ namespace ErrorCodes
     extern const int LOGICAL_ERROR;
 }
 
-std::pair<String, Field> ConditionSelectivityEstimator::extractBinaryOp(const RPNBuilderTreeNode & node, const String & column_name) const
+void ConditionSelectivityEstimator::extractOperators(const RPNBuilderTreeNode & node, const String & qualified_column_name, std::vector<std::pair<String, Field>> & result) const
```

Comment:
I think this needs a code comment what kind of predicate is supported and what is returned.